Tasks/Common/PowershellHelpers/Helpers.ps1

```diff
@@ -19,7 +19,10 @@ function Invoke-ActionWithRetries
         $RetryableExceptions,
 
         [string]
-        $RetryMessage
+        $RetryMessage,
+
+        [scriptblock]
+        $OnException = { $true }
```

Comment:
Consider renaming like ShouldRetryHandler

Reply:
Renamed to ShouldRetryOnException